src/main/java/com/eharmony/pho/api/DataStoreApi.java

```diff
@@ -72,4 +74,13 @@
      */
     <T, R> R findOne(QuerySelect<T, R> query);
 
+    /**
+     * Save selective field for the given entity
+     * 
+     * @param entity
+     * @param selectedFields
+     * @return
+     */
+    <T> T save(T entity, List<String> selectedFields);
```

Comment:
underlying QuerySelectImpl supports List. Being consistent with that and not having to convert from Set to List.